fitbenchmarking/fitbenchmarking
# Add LMFIT software #1096

#### Description of Work

Fixes #1096


#### Testing Instructions

1. Check the the packages can be installed correctly using setup.py
2. Check that the minimizer works as expected
3. Check the documentation for any error e.g. licensing etc

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
